.github/workflows/test-cli.yaml

```diff
@@ -0,0 +1,52 @@
+name: Test_CLI
+
+on:
+  pull_request:
+    paths:
+      - "cli/src/**"
+      - "cli/test/**"
+      - "cli/package*.json"
+      - "cli/tsconfig*.json"
+      - "cli/jest.config.js"
+  push:
+    branches:
+      - main
+    paths:
+      - "cli/src/**"
+      - "cli/test/**"
+      - "cli/package*.json"
+      - "cli/tsconfig*.json"
+      - "cli/jest.config.js"
+
+# env:
+# NODE_VERSION: "20.17.0" # Fixed version for better stability
+
+jobs:
+  main:
+    runs-on: token-studio-linux-medium
+    container:
+      image: node:20.17.0-alpine3.20 # Using alpine for a smaller image
+    steps:
+      # * Initial steps
+      - name: Checkout repository
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29
```

Comment:
Add specific version as # after the commit hash.

Reply:
version added